compiler/rustc_builtin_macros/src/autodiff.rs

```diff
@@ -562,7 +561,7 @@ fn gen_enzyme_decl(
                 let old_name = if let PatKind::Ident(_, ident, _) = arg.pat.kind {
                     ident.name
                 } else {
-                    trace!("{:#?}", &shadow_arg.pat);
+                    dbg!(&shadow_arg.pat);
```

Comment:
Problem [DBG 1/n]: this should not be `dbg!` because we need to guard debug logging via tracing. This could be `debug!`.